Calling/App.xaml.cs

```diff
@@ -23,31 +36,59 @@ public App()
         }
 
         /// <summary>
-        /// Invoked when the application is launched normally by the end user.  Other entry points
-        /// will be used such as when the application is launched to open a specific file.
+        // Handle protocol activations.
         /// </summary>
-        /// <param name="e">Details about the launch request and process.</param>
-        protected override void OnLaunched(LaunchActivatedEventArgs e)
+        protected override async void OnActivated(IActivatedEventArgs e)
         {
-            Frame rootFrame = Window.Current.Content as Frame;
+            await InitNotificationsAsync();
 
-            // Do not repeat app initialization when the Window already has content,
-            // just ensure that the window is active
-            if (rootFrame == null)
+            if (e.Kind == ActivationKind.Protocol || e is ToastNotificationActivatedEventArgs)
             {
-                // Create a Frame to act as the navigation context and navigate to the first page
-                rootFrame = new Frame();
+                Frame rootFrame = CreateRootFrame();
```

Comment:
Consistency: please use `var` whenever you can in this sample code.

Reply:
Not needed anymore, the code to get the frame is Visual Studio generated code when the project is created.